assets/aml-benchmark/components/src/aml_benchmark/batch_inference_preparer/endpoint_data_preparer.py

```diff
@@ -32,10 +36,14 @@ def convert_ground_truth(
     ) -> Dict[str, Any]:
         """Convert the ground truth to the corresponding payload with id."""
         row_id = EndpointUtilities.hash_payload_prompt(payload, self._model)
-        return {
+        result = {
             EndpointDataPreparer.PAYLOAD_HASH: row_id,
             EndpointDataPreparer.PAYLOAD_GROUNDTRUTH: origin_json_dict.get(self._label_key, ""),
         }
+        if self._additional_columns:
+            for column in self._additional_columns:
+                result[column] = origin_json_dict.get(column, "")
```

Comment:
Can we add a user exception if the column does not exist?

Reply:
would we really want to throw an exception or just log this? If we just log `f"Skipping adding additional column {additional_column} as this column does not exist."` would bring down the failure rate.